packages/dds/tree/src/test/shared-tree/fuzz/topLevelFuzz.spec.ts

```diff
@@ -55,11 +60,26 @@ describe("Fuzz - Top-Level", () => {
 	const random = makeRandom(0);
 	const runsPerBatch = 20;
 	const opsPerRun = 20;
+	const editGeneratorOpWeights: EditGeneratorOpWeights = {
+		insert: 5,
+		delete: 1,
```

Comment:
This configurable weight thing just gave me an idea: instead of skipping the fuzz test, we could just configure it to not generate any inserts or deletes. (Same for the test in targetedFuzz.ts)!

Reply:
Sounds good! I have now removed the skip in the `targetedFuzz.ts`, but the `toplevelFuzz.ts` are still failing due to the synchronize operation. Should I also apply a weight to the synchronize operation to have the option of turning this off as well?

I also noticed that there is another error sometimes when abort is called `0x4a6 /* Field not present in sparse node */` in `getDescendant()` of `sparseTree.ts`. Should I remove the abort op in this file for now as well? Or keep the top level tests skipped for now?

Reply:
Actually, it makes sense that the synchronize operation fails now, because some of the changes on the tree will have been put in a transaction but not committed.

There's several things we could do:
* prevent the synchronize operation from being generated when one of the trees has a transaction pending
* commit any pending transaction when applying the synchronize operation
* abort any pending transaction when applying the synchronize operation
* set the "start", "abort", and "commit" weights to 0
* instead of using the "start", "abort", and "commit" transaction APIs, we could use the branching APIs so that we can access the state of the tree that doesn't reflect uncommitted local changes, and use that in our comparison of the trees during the synchronization check.

I think the last option is the best, but that's the most annoying one to implement so I think for now you can just set the "start", "abort", and "commit" weights to 0 (meaning deleting the config entries for them).

Reply:
If you still get error 0x4a6 after that then keep the test skipped.

Reply:
set the "start", "abort", and "commit" weights to 0